debezium-core/src/test/java/io/debezium/relational/TableSchemaBuilderTest.java

```diff
@@ -297,17 +298,60 @@ public void shouldSanitizeFieldNamesAndBuildTableSchemaFromTableWithoutPrimaryKe
         assertThat(value.get("C5")).isEqualTo(ByteBuffer.wrap(new byte[]{ 71, 117, 110, 110, 97, 114}));
         assertThat(value.get("C6")).isEqualTo(Short.valueOf((short) 0));
     }
-
+    
     @Test
     @FixFor("DBZ-1044")
     public void shouldSanitizeFieldNamesAndValidateSerialization() {
         schema = new TableSchemaBuilder(new JdbcValueConverters(), adjuster, SchemaBuilder.struct().build(), true)
-                .create(prefix, "sometopic", table, null, null);
+                .create(prefix, "sometopic", table, null, null, null);
 
         Struct key = (Struct) schema.keyFromColumnData(keyData);
         Struct value = schema.valueFromColumnData(data);
 
         SourceRecord record = new SourceRecord(Collections.emptyMap(), Collections.emptyMap(), "sometopic", schema.keySchema(), key, schema.valueSchema(), value);
         VerifyRecord.isValid(record);
     }
+
+    @Test
+    @FixFor("DBZ-1015")
+    public void shouldBuildTableSchemaFromTableWithCustomKey() {
```

Comment:
Is there a test somewhere for configuring multiple tables?

Reply:
test added

Reply:
Thx!